package.json

```diff
@@ -304,11 +305,16 @@
         "@types/vscode": "1.57.0",
         "@typescript-eslint/eslint-plugin": "^4.28.3",
         "@vscode/test-electron": "^2.2.3",
+        "assert": "^2.0.0",
+        "browserify-zlib": "^0.2.0",
+        "console-browserify": "^1.2.0",
```

Comment:
I'll have to re-visit if all these devDependencies are necessary, but since they're dev-- at least they shouldn't be shipped with the extension.